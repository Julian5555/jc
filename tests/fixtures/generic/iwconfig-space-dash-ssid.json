[{"name":"wlp-50","protocol":"IEEE 802.11","essid":"BLABLABLA-bla bla bla","mode":"Managed","frequency":5.18,"frequency_unit":"GHz","access_point":"E6:63:DA:16:50:BF","bit_rate":6.0,"bit_rate_unit":"Mb/s","tx_power":30,"tx_power_unit":"dBm","retry_short_limit":7,"rts_threshold":false,"fragment_threshold":false,"power_management":true,"link_quality":"58/70","signal_level":-52,"signal_level_unit":"dBm","rx_invalid_nwid":0,"rx_invalid_crypt":0,"rx_invalid_frag":0,"tx_excessive_retries":0,"invalid_misc":1766,"missed_beacon":0}]
