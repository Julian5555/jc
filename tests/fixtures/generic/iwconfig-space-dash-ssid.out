wlp-50    IEEE 802.11  ESSID:"BLABLABLA-bla bla bla"  
          Mode:Managed  Frequency:5.18 GHz  Access Point: E6:63:DA:16:50:BF   
          Bit Rate=6 Mb/s   Tx-Power=30 dBm   
          Retry short limit:7   RTS thr:off   Fragment thr:off
          Power Management:on
          Link Quality=58/70  Signal level=-52 dBm  
          Rx invalid nwid:0  Rx invalid crypt:0  Rx invalid frag:0
          Tx excessive retries:0  Invalid misc:1766   Missed beacon:0

